examples/spouts/spout101/README.md

```diff
@@ -0,0 +1,181 @@
+# Spout Pipelines 
+>![pach_logo](./img/pach_logo.svg) This new implementation of the spout functionality is available in version **1.12 and higher**.
+
+
+## Intro
+A spout is a type of pipeline that ingests 
+streaming data (message queue, database transactions logs,
+event notifications... ). 
+See spout pipelines as a bridge
+between an external stream of data and Pachyderm's repo.
+In Martin Fowler's enterprise integration patterns' terms,
+a Pachyderm spout is a 
+*[Polling Consumer](https://www.enterpriseintegrationpatterns.com/patterns/messaging/PollingConsumer.html)* 
+(actively polls for a message, processes it, then polls for another).
+
+Generally, you use spouts for situations 
+when the interval between new data generation
+is large or sporadic, but the latency requirement
+to start the processing is short. 
```

Comment:
```suggestion
Generally, spout pipelines are ideal for situations 
when the frequency of new incoming data 
is sporadic and the latency requirement
to start the processing is short. 
```

Reply:
I don't think of spouts as particularly suited or not for large time intervals between data inputs. They are totally fine for this, but it's actually just as common (or sometimes moreso) use spouts for streaming data that might be coming in at a very high frequency. 

Do you think it's worth making a not here (or somewhere, I haven't finished reading the whole docs yet) that users should be careful not to use spouts to make "too many" commits too frequently. While small, there is an overhead per commit and per job (sophisticated versioning and lineage tracking aren't free unfortunately) and therefore we generally recommend making commits no more frequently that a few per minute. Pachyderm in general can handle micro batches much better that full streaming.